app/Http/Controllers/StickerController.php

```diff
@@ -2,64 +2,84 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Pack;
 use App\Models\Sticker;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Validation\Rules\File;
+
+use function PHPUnit\Framework\stringStartsWith;
 
 class StickerController extends Controller
 {
     /**
      * Display a listing of the resource.
      */
-    public function index()
-    {
-        //
-    }
-
-    /**
-     * Show the form for creating a new resource.
-     */
-    public function create()
+    public function index(Pack $pack)
     {
         //
+        return $pack->stickers;
```

Comment:
Use json responses!

Reply:
laravel is atuomaticly convert model to json response so it will return like thi
{
  "image": <image_path>,
  "pack_id": <pack_id>
}

source: [link](https://laraveldaily.com/post/how-to-automatically-return-json-for-laravel-eloquent#:~:text=By%20default%2C%20Laravel%20transforms%20the,a%20Model%20or%20Eloquent%20Collection.)

Reply:
Please use the following format
1. If successful 
```{"status": bool, "data": dict}```
2. If it fails 
```{"status": bool, "message": string}```

Reply:
Why do you have to follow the standards? because it will make it easier to process data